front-end/src/components/cookie/CookieView.js

```diff
@@ -30,18 +29,18 @@ class CookieView extends Component {
         const showCookieDiv = !this.isCookieSet(cookiePolicy) && this.state.show;
 
         return (
-            <Modal show={showCookieDiv} onHide={this.handleClose} bsClass="cookie-modal">
+            <Modal show={showCookieDiv} onHide={this.handleClose} dialogClassName={styles["cookie-modal"]} backdropClassName={styles["cookie-modal-backdrop"]}>
                 <Modal.Header closeButton>
```

Comment:
this seems like boolean.
Please name it something likes: `hasCloseButton`

Reply:
This is an internal prop implemented by react-bootstrap. Need to be named `closeButton` according to their docs